.github/scripts/tests/execute-tests.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
I am not sure, for what this file is introduced.

Reply:
It allows executing the tests from WSL command line, including proper setting of host `PATH` so the toolchain DLLs are used.